src/CloudGraphsDFG/services/CloudGraphsDFG.jl

```diff
@@ -216,13 +227,12 @@ function mergeUpdateVariableSolverData!(dfg::CloudGraphsDFG, sourceVariable::DFG
     if !exists(dfg, sourceVariable)
         error("Source variable '$(sourceVariable.label)' doesn't exist in the graph.")
     end
-    var = getVariable(dfg, sourceVariable.label)
-    newEsts = merge!(var.ppeDict, deepcopy(sourceVariable.ppeDict))
-    newSolveData = merge!(var.solverDataDict, sourceVariable.solverDataDict)
-    Neo4j.setnodeproperty(dfg.neo4jInstance.graph, var._internalId, "ppeDict",
-        JSON2.write(newEsts))
-    Neo4j.setnodeproperty(dfg.neo4jInstance.graph, var._internalId, "solverDataDict",
-        JSON2.write(Dict(keys(newSolveData) .=> map(vnd -> packVariableNodeData(dfg, vnd), values(newSolveData)))))
+    for (k,v) in sourceVariable.ppeDict
+        updatePPE!(dfg, getLabel(sourceVariable), v, k)
+    end
+    for (k,v) in sourceVariable.solverDataDict
+        updateVariableSolverData!(dfg, getLabel(sourceVariable), v, k)
+    end
```

Comment:
I also made a helper to do both of these in one go:
`mergeVariableData!` https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/3301d4fef1fdcd75962b2e7fc1ac21a320eb5751/src/services/AbstractDFG.jl#L909